ffmeta/data/ffmetadata_ddl.sql

```diff
@@ -60,122 +60,131 @@ CREATE TABLE IF NOT EXISTS `raw2` (
   `wave` char(30) CHARACTER SET latin1 DEFAULT NULL,
   `section` char(1) DEFAULT NULL,
   `leaf` char(30) DEFAULT NULL,
+  `obs` int(4) DEFAULT NULL,
+  `min` int(7) DEFAULT NULL,
+  `max` int(20) DEFAULT NULL,
+  `avg` float(7) DEFAULT NULL,
+  `std` float(7) DEFAULT NULL,
   `value1` varchar(20) DEFAULT NULL,
   `label1` varchar(200) DEFAULT NULL,
+  `freq1` int(4) DEFAULT NULL,
+  `per1` float(4) DEFAULT NULL,
   `value2` varchar(20) DEFAULT NULL,
   `label2` varchar(200) DEFAULT NULL,
+  `freq2` int(4) DEFAULT NULL,
```

Comment:
for `freqN` fields, a display width of 4 might be too restrictive, probably go higher than you think you need to save some headaches in the future?